### PR TITLE
Improve documentation about how to run the extension

### PR DESCRIPTION
**Pull request by individual-it**
on top of #6

### CODE REVIEW

**Comment by grgprarup** [comment]
@individual-it PR #6 is merged you can rebase and make pr ready for review.

File: web.config.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      "config": {
+        "mimeTypes": ["image/tiff", "image/bmp", "image/x-ms-bmp"]
+      }
+    },
+    {
+      "id": "presentation-viewer",
+      "path": "https://<url-where-the-dist-folder-is-served>/web-app-presentation-viewer.js"
+    }
+  ]
+}
```
Comment (grgprarup) [inline web.config.json:54]: Are we adding this config file at the root of project?

Comment (individual-it) [reply web.config.json:54]: we can put it also somewhere else, but need to write the docs accordingly
where would you want to have it?

**Review by phil-davis — APPROVED** [review]
Looks reasonable to me.
I suppose that `web.config.json` goes in the top-level of the repo so that it is easily found when running for local development etc.

**Comment by koebel** [comment]
From my understanding, `web.config.json` is just some sample code. If that is the case, please treat it in the same fashion as this is done in ocis web, see also my comment on this subject here: https://github.com/JankariTech/web-app-presentation-viewer/pull/19#issuecomment-1945446544

**Comment by individual-it** [comment]
@grgprarup @koebel I've moved the config file and adjusted the docu  accordingly. Please review again

File: README.md [hunk 2]
```diff
@@ -21,15 +21,44 @@ pnpm build
 ```
 The extension will be built in the `dist` directory.
 
+### 3. Serve the extension
+1. serve the content of the `dist` folder using any HTTP web-server
+2. configure web
+   1. if you have already a dedicated `web.config.json` file, add a `external_apps` section (or edit the existing one). It has to have an item with `"id": "presentation-viewer"` and a `path` pointing to `web-app-presentation-viewer.js` e.g.:
```
Comment (grgprarup) [inline README.md:27]:
```suggestion
   1. If you already have a dedicated `web.config.json` file, add an `external_apps` section (or edit the existing one). It has to have an item with `"id": "presentation-viewer"` and a `path` pointing to `web-app-presentation-viewer.js` e.g.:
```

**Review by grgprarup — APPROVED** [review]

**Pull request merged**